yaksh/models.py

```diff
@@ -402,6 +402,9 @@ def read_json(self, file_path, user, files=None):
             with open(json_file, 'r') as q_file:
                 questions_list = q_file.read()
                 self.load_questions(questions_list, user, file_path, files)
+            return "Questions Uploaded Successfully"
+        else:
+            return "Please upload zip file with questions_dump.json in it."
```

Comment:
We are not raising an exception if the json data in the file is syntactically wrong. In my testing, uploading an improper json file will raise an ValueError exception with the information of where in the json file it found an error. Maybe we can display the error with its location.